### PR TITLE
ovn-setup.sh changes

### PR DESCRIPTION
**Pull request by pecameron**
Wait for cluster and master-api to come up
use /etc/origin/master/master-config.yaml as the source
of the net_cidr and svc_cidr

Signed-off-by: Phil Cameron <pcameron@redhat.com>

### CODE REVIEW

**Comment by pecameron** [comment]
@dcbw @squeed @rajatchopra @glenntwest PTAL

File: dist/ansible/scripts/ovn-setup.sh [hunk 1]
```diff
@@ -10,23 +10,77 @@
...
+#  - cidr: 10.128.0.0/14
+#    hostSubnetLength: 9
+#    externalIPNetworkCIDRs:
+#  - 0.0.0.0/0
+#    networkPluginName: cni
+#  serviceNetworkCIDR: 172.30.0.0/16
+
 # get the network cidr (master-config.yaml clusterNetworkCIDR)
-net_cidr=10.128.0.0/14
+cfg=/etc/origin/master/master-config.yaml
```
Comment (rajatchopra) [inline dist/ansible/scripts/ovn-setup.sh]: Can we take this path as an argument to the script (and just keep this as a default maybe) ?
Also, if one cannot obtain the cidr/sub values, can we resort to a default?

Comment (pecameron) [reply dist/ansible/scripts/ovn-setup.sh]: @rajatchopra done. PTAL

File: dist/ansible/scripts/ovn-setup.sh [hunk 2]
```diff
@@ -10,23 +10,84 @@
...
-if [ $health != 'ok' ]
-then
-  echo " cluster not ready"
-  exit 1
-fi
+trap 'kill $(jobs -p); exit 0' TERM
+
+cfg=${1:-"/etc/origin/master/master-config.yaml"}
+
+wait_for_cluster_up () {
```
Comment (dcbw) [inline dist/ansible/scripts/ovn-setup.sh]: These two wait functions basically do the same thing except for the specific command they run, perhaps we should just steal wait-for-condition() from OpenShift's images/dind/node/openshift-dind-lib.sh?

Comment (pecameron) [reply dist/ansible/scripts/ovn-setup.sh]: @dcbw, I will take a look.

**Comment by pecameron** [comment]
@dcbw refactored script for common wait function. PTAL

**Comment by squeed** [comment]
Seems reasonable. Someday we'll want to plug this through with configmaps and/or environment variable,s but not now.

**Comment by pecameron** [comment]
@dcbw can you merge this?

**Pull request merged**